.github/workflows/bump.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+"""
+Determine next version number for versions of a schema like v1.0
+based on existing git tags and which component to bump (minor/major).
+"""
+
+import subprocess
+import re
+import sys
+import os
+
+
+def convert_version_to_sortable_int(major, minor):
+    return major * 1000 + minor
+
+
+def determine_most_recent_existing_version():
+    tags = subprocess.run(["git", "tag"], capture_output=True).stdout.splitlines()
+
+    versions = []
+
+    for t in tags:
+        tag = t.decode()
+        if re.match(r"v[0-9]+.[0-9]+", tag):
+            tag_without_prefix = tag[1:]
+            components = tag_without_prefix.split(".")
+            assert len(components) == 2
+            major_int = int(components[0])
+            minor_int = int(components[1])
+            versions.append(
+                {
+                    "tag": tag,
+                    "sortNumber": convert_version_to_sortable_int(major_int, minor_int),
+                    "major": major_int,
+                    "minor": minor_int,
+                }
+            )
+
+    if len(versions) == 0:
+        print("No existing versions found")
+        return {
+            "tag": "v0.0",
```

Comment:
We already have some versions, so we would probably not hit this case 

⚠️ 🚲  😄

Reply:
I've put this in on purpose to deal with the fact that we don't have any releases so far except for 'latest' which is not a version number. Am I missing something or are we talking about different things?

<img width="628" alt="Screenshot 2023-12-18 at 14 24 18" src="https://github.com/gardenlinux/builder/assets/2292245/9dd00321-4875-4476-92c1-0697757a7b24">

Reply:
It's returning v0.0 on purpose because it will be increased either by major or minor, depending on the parameter.

So the first version will either be v1.0 or v0.1 depending on the parameter. I think that's reasonable and the expected behaviour.

Reply:
Ah ok, did not understand it then. Thanks for explaining.